src/crypto/tls/cfkem.go

```diff
@@ -0,0 +1,88 @@
+// Glue to add Circl's (post-quantum) hybrid KEMs to quic-go.
```

Comment:
quic-go, hmmm...

Reply:
Still says `quic-go` here and `qtls.X25519` below. Can we update this for our normal crypto/tls fork?